dev-tools/packer/Makefile

```diff
@@ -1,28 +1,36 @@
-DATE:=$(shell date +%y%m%d%H%M%S)
-BUILDID?=$(DATE)
-
+BUILDID=$(shell git rev-parse --short HEAD)
```

Comment:
If this is only being used for the `.build_hash.txt` file then I don't think it needs to be abbreviated (i.e. `--short`).